.gitpod.yml

```diff
@@ -0,0 +1,16 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/config-gitpod-file)
+# and commit this file to your remote git repository to share the goodness with others.
+image:
+  file: .gitpod/Dockerfile
+  
+tasks:
+  - init: ./mvnw install -DskipTests=false
```

Comment:
Should it be `./mvnw -Dquickly` instead? I'm not exactly sure when this task is executed?

Reply:
This command was already there. I didn't want to change it.

Reply:
but I can change it this command is better.

Reply:
Already where? This is a new file you are adding :).

Reply:
🤔 I didn't remember.. I changed that.